Traceback (most recent call last):
  File "/home/local/ASURITE/nnjungle/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 447, in compile
    self._evaluate("Compiling", inputset)
  File "/home/local/ASURITE/nnjungle/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 292, in _evaluate
    self._trace(first_sample)
  File "/home/local/ASURITE/nnjungle/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 220, in _trace
    self.graph = Tracer.trace(self.function, parameters)
  File "/home/local/ASURITE/nnjungle/.local/lib/python3.10/site-packages/concrete/fhe/tracing/tracer.py", line 81, in trace
    output_tracers: Any = function(**arguments)
  File "/tmp/ipykernel_417971/1871905759.py", line 3, in add
    sum =  x[0][0][0][0] + x[0][0][0][1]
  File "/home/local/ASURITE/nnjungle/.local/lib/python3.10/site-packages/concrete/fhe/tracing/tracer.py", line 803, in __getitem__
    output_value.shape = np.zeros(output_value.shape)[index].shape  # type: ignore
IndexError: too many indices for array: array is 0-dimensional, but 1 were indexed
